docs/compatibility.md

```diff
@@ -560,3 +560,11 @@ double quotes around strings in JSON data, whereas Spark allows single quotes ar
 data.  The RAPIDS Spark `get_json_object` operation on the GPU will return `None` in PySpark or
 `Null` in Scala when trying to match a string surrounded by single quotes.  This behavior will be
 updated in a future release to more closely match Spark.
+
+## Approximate Percentile
+
+The `approximate_percentile` operation is now supported on the GPU. The GPU implementation uses
```

Comment:
nit: I think we should reword this a bit.

The `approximate_percentile` operation on the GPU uses ...